### DISCUSSION TITLE
2023-12-01 - 📱 SSW Rewards Engagement - Chewing the Fat 📈

### DISCUSSION THREAD

**Discussion by bradystroud** [SSWConsulting/SSW.Rules.Content#7378] [Chewing the Fat]
Hey SSWers,

Lets talk about SSW Rewards!

**Comment by MarcusIrmscherSSW** [comment]
Great Video Matt, and some awesome ideas for the App 😀. It will be nice to see the Rewards app get an overhaul. 🔨

**Comment by matt-goldman** [comment]
Looking forward to seeing everyone's thoughts on this. Keen to make SSW Rewards awesome and would love for more people to get involved.

**Comment by jakebayliss** [comment]
Some really cool ideas here with the prizes and leaderboard changes :)

**Comment by drwharris** [comment]
Gamify everything. Quizzes are great but you need different styles of "game" to keep people engaged.

**Comment by alexleessw** [comment]
Looking forward to using this :)

**Comment by cvionnet** [comment]
As always, very good presentation and explanations Matt.
❤️The energy pushed into improving the app and bringing lot of great ideas!

**Comment by joshbermanssw** [comment]
Great presentation in the video - will be interesting to see what comes out of the app. Alot of cool stuff planned.

**Comment by TobyChurches** [comment]
Matt is a excellent presenter! Very clear and easy to listen to
Heaps of great ideas! looking forward to seeing what the app becomes 😁

**Comment by amankumarrr** [comment]
I enjoyed the video and found the ideas presented very compelling. I believe implementing these concepts could significantly improve the usability of the Reward App.

**Comment by Harry-Ross** [comment]
Super engaging and important video for the future of SSW Rewards. A really great watch - seeing the thought processes behind why certain features have been proposed is really interesting to see :)

**Comment by AntPolkanov** [comment]
So many great features! Some of them are not gonna be easy to implement, but it's absolutely worth the effort!

**Comment by leehawkinsssw** [comment]
I like the approach taken with this video - explaining why changes are required and then suggesting ways to move the needle in the direction we want for this app. 

The video was too long for a CTF (at least for me on client work) so I didn't get to enjoy all of Matt's no doubt excellent ideas.

The Google Trends data shown in the rule is not at all compelling.

**Comment by jeoffreyfischer** [comment]
Love most of the ideas! But please no more 30min long videos 🥲

**Comment by chrisschultzssw** [comment]
Lots of potential in the Rewards app - looking forward to seeing these improvements!

**Comment by Calinator444** [comment]
I created a GitHub discussion talking about my exposure to the app. Please comment/discuss.

https://github.com/SSWConsulting/SSW.Rewards.Mobile/discussions/498

**Comment by vladislav-kir** [comment]
Amazing video from Matt! 🤩 
I only wish I had time to watch it all right now...

**Comment by Levijj22** [comment]
Great video Matt!
Let's make this app modern!

**Comment by AshAnil** [comment]
Nice video Matt 😍 I liked the **Recruitment via the SSWRewards app** feature the most.

**Comment by calumjs** [comment]
Looking forward to figuring out how we can shoehorn GPT into SSW Rewards

**Reply by matt-goldman** [reply]
I actually have a plan for this...

**Comment by sethdaily** [comment]
I think if we're going to push it at events and conferences it needs to be very simple and very easy to use.

**Comment by steven0x51** [comment]
Great video!

**Comment by UlyssesMaclaren** [comment]
This mobile app should have 2 functions:
1. show people at conferences that we can make a modern mobile app with good UX
2. At that conference, allow them to compete for a prize draw, and help them get more involved with us at the event via gamification

I think it's important we don't do anything that doesn't achieve the above... it should not be aimed at anyone outside of that event. Otherwise we're going to be flogging a dead horse which would be expensive and pointless

**Comment by synnoveisabel** [comment]
Great video! Love the ideas on improving the UI

**Comment by JackDevAU** [comment]
Loved the video and the planned features! 
Keen to see it come to life!

**Comment by bettybondoc** [comment]
I can see why Adam said Matt is a great consultant — he sure is convincing! Great video. 🤩

I haven't finished it yet though cause I got sidetracked when we came to the UI discussion part and I spent my timeboxed time looking for articles on parallax — I left a comment in the GitHub issue.

**Comment by isaaclombardssw** [comment]
Really interesting prospective features to showcase what SSW mobile can do.

**Comment by zacharykeeping** [comment]
Really good video - will be good to see these features implemented!

**Comment by ricksu978** [comment]
Fantastic presentation, Matt! Your insights and proposed features for the Rewards App are really inspiring. It's exciting to think about the potential transformations and how they'll enhance user engagement. Looking forward to the next steps in this journey! 🚀

**Comment by tomek-i** [comment]
Good video, awesome ideas just weekly engagement could be difficult to achieve unless.
looking forward to the grand prices :-D

**Comment by CoffeeBugHub** [comment]
Great video easily engaging the audience. 
Looking forward to seeing a shiny new rewards app.

**Comment by jimmy1992abc** [comment]
Great video, but some typos in the rule.
I am wondering in which scenario people would use this App and what the App can bring to SSW.
Can we make SSW Reward an App that pushes tech news or tech related staff, and can be a hub for developers, people are able to claim rewards for the use of the app. Traffic is the most valuable asset.

**Comment by jackreimers** [comment]
Great video, looking forward to seeing the improvements!

**Comment by christoment** [comment]
I like the idea that SSW Rewards as the showcase of SSW's design - I think we should put a bit effort on this, distill it, and then apply the learnings to our other producs (website, SugarLearning, TimePro) so we can have same, unified polished experience.

**Comment by JackDuan123** [comment]
Mobile App is dead. It will take on average $1 to gain a user for mobile app and the retention rate is probably less than 30%, so in order to run a mobile app, on average the App owner needs to earn more than $3 per user...

It's almost a mission of impossible.

**Comment by bahjatalaadelssw** [comment]
Can wait for the UI and proper points tracking!

**Comment by Ben0189** [comment]
Great video from Matt G, i like the #1 social media integrations idea🥂

**Comment by SamBWagner** [comment]
Video was god tier. I don't really think that the Rewards app is a cow worth milking but I'm not aware of all the metrics an other options available in the mobile space

**Comment by stephanfakossw** [comment]
Matt G is an amazing presenter, and the idea isn't bad... However, while this is cool I do not see any sort of priority in this!

**Comment by PennyWalker** [comment]
Great video Matt - super cool and some awesome ideas. I'd love to see more outside engagement with the app. It feels like such a waste to only use it for conferences so anything that kicks up engagement is a win. It'll be awesome to showcase your skills too! :)

**Comment by Marxoz** [comment]
Loved the video Matt. Also like the voting and commenting on the different ideas.

**Comment by Aibono1225** [comment]
Great video with excellent ideas for improving our SSW Rewards. I also love the presentation style in the video, it's really convincing!

**Comment by KristenHu** [comment]
Great video Matt! Glad to see the vote, and I like the idea of social media🎉

**Comment by babakamyljanovssw** [comment]
Great video!!! I loved the idea about integration of 3D feature into app.

**Comment by AnkyChe** [comment]
Nice video, some ideas are awesome! I love the ideal of  Prize draw: filtering based on completed achievements.

**Comment by sylhuang** [comment]
I love the video and the features we are going to build are going to make this app so much more interesting. Already getting excited about it lol

**Comment by jimmidier** [comment]
Great video and admire your work Matt!

**Comment by tombui99** [comment]
This is an awesome video, these ideas are all awesome .Thanks for the huge effort Matt!

**Comment by PothieuG** [comment]
Awesome video Matt ! Well explained.
Thanks for the work, it's very inspiring  :)

**Comment by BrookJeynes** [comment]
Great video, seems like there's some interesting and new features coming soon.
